eng/pipelines/linux.yml

```diff
@@ -115,14 +125,6 @@ jobs:
               _buildScriptPrefix: 'ROOTFS_DIR=/crossrootfs/arm64 '
               _buildExtraArguments: -warnAsError false /p:BuildNativeClang=--clang5.0 /p:RuntimeOS=linux-musl
 
-            arm_Release:
```

Comment:
Can we then leave this for when `isOfficialBuild == false` -- we don't want to drop guarding the arm32 vertical. Let's keep building it without running tests for CI then.

Reply:
I wanted to put that back but I forget to do it in final cleanup.